src/bindings/python/flux/accounting/test/test_user_subcommands.py

```diff
@@ -116,6 +116,23 @@ def test_05_edit_bad_field(self):
 
         self.assertRaises(ValueError)
 
+    # delete a user from the association table
+    def test_06_delete_user(self):
+        cursor = acct_conn.cursor()
+        cursor.execute(
+            "SELECT * FROM association_table WHERE username='fluxuser' AND bank='acct'"
+        )
+        num_rows_before_delete = cursor.fetchall()
+
+        aclif.delete_user(acct_conn, user="fluxuser", bank="acct")
+
+        cursor.execute(
+            "SELECT * FROM association_table WHERE username='fluxuser' AND bank='acct'"
+        )
+        num_rows_after_delete = cursor.fetchall()
+
+        self.assertGreater(len(num_rows_before_delete), len(num_rows_after_delete))
```

Comment:
test could be more accurate if you you calculated the exact number of rows you expect to delete?  And make sure that the exact number is deleted?

Reply:
Oh okay, I think I see what you're saying. Yeah, I could test to make sure the length of the returned list is an exact number. I just pushed up a commit that performs two checks on the `SELECT` query before and after the user gets deleted. Before, it should return a length of 1, and afterwards, it should return a length of 0.